docs/resources/Guild.md

```diff
@@ -1347,3 +1366,16 @@ There are currently several caveats for this endpoint:
 - You must have the `MUTE_MEMBERS` permission. (Since suppression is the only thing that is available currently.)
 - When unsuppressed, non-bot users will have their `request_to_speak_timestamp` set to the current time. Bot users will not.
 - When suppressed, the user will have their `request_to_speak_timestamp` removed.
+
+## Modify Guild Incident Actions % PUT /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/incident-actions
+
+Modifies the incident actions of the guild. Returns a 200 with the [Incidents Data](#DOCS_RESOURCES_GUILD/incidents-data-object) object for the guild. Requires the `MANAGE_GUILD` permission.
+
+###### JSON Params
+
+| Field                   | Type                  | Description                                                              |
+|-------------------------|-----------------------|--------------------------------------------------------------------------|
+| invites_disabled_until? | ?ISO8601 timestamp \* | when disabled invites will expire (up to 24 hours in the future)         |
+| dms_disabled_until?     | ?ISO8601 timestamp \* | when disabled direct messages will expire (up to 24 hours in the future) |
```

Comment:
It'd be better to simply reuse the incident data object's description here (with the added 24h constraint), especially since this one sounds a bit weird imo.

Reply:
done